### ISSUE TITLE
Rename wrong variable naming idWorkspace and idContent in entire project

### ISSUE THREAD

**Issue by Skylsmoi** [tracim/tracim#1887]
## Bug description & symptom

A wrong naming usage for idWorkspace and idContent exists in Tracim.
It should be renamed respectively workspaceId and contentId.

## Diagnostic

This refactor will take place in many components of frontend and some apps

## Expected

Naming must be `workspaceId` and `contentId`

## Impacted components

Mostly frontend and at least:
- app html-doc
- app file
- app thread

**Comment by Skylsmoi** [comment]
the PR has been merged.
Since this refactor apply to the whole frontend, it will be nice to have a proper test phase before next Tracim release.
But only after this PR (https://github.com/tracim/tracim/issues/1956) is merged too because it also apply to the whole frontend. Although this one is a lot safer in term of regressions

**Comment by PhilippeAccorsi** [comment]
I have tested many possibility in frontend and everything work good.
For me its verified.
